suite-common/wallet-core/src/send/sendFormEthereumThunks.ts

```diff
@@ -227,10 +239,16 @@ export const composeEthereumTransactionFeeLevelsThunk = createThunk<
                     ? customFeeLimit.toFixed(0)
                     : undefined;
             }
-            if (tx.type === 'error' && tx.error === 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE') {
+            if (
+                tx.type === 'error' &&
+                tx.error === 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE_WITH_ETH_AMOUNT'
+            ) {
                 tx.errorMessage = {
-                    id: 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE',
-                    values: { symbol: network.symbol.toUpperCase() },
+                    id: 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE_WITH_ETH_AMOUNT',
+                    values: {
+                        symbol: tx.errorMessage?.values?.symbol || 'ETH',
```

Comment:
use `symbol` from account, never hardcode symbols in generic evm code

Reply:
https://github.com/trezor/trezor-suite/pull/15993/commits/0d79d6dff1cd5b390161fc528e10e65153248453